pkg/client/kubernetes/base/hpa.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetesbase
+
+// DeleteHPA deletes an already existing HPA object.
+func (c *Client) DeleteHPA(namespace, name string) error {
+	return c.clientset.Autoscaling().HorizontalPodAutoscalers(namespace).Delete(name, &defaultDeleteOptions)
```

Comment:
Using `Autoscaling()` is deprecated: "Deprecated: Autoscaling retrieves the default version of AutoscalingClient. Please explicitly pick a version." -> Please use `V2beta1()`.